pkg/server/api/agent/v1/service_test.go

```diff
@@ -2,26 +2,128 @@ package agent_test
 
 import (
 	"context"
+	"errors"
 	"testing"
 
 	"github.com/sirupsen/logrus/hooks/test"
 	"github.com/spiffe/spire/pkg/server/api/agent/v1"
 	"github.com/spiffe/spire/pkg/server/api/rpccontext"
 	agentpb "github.com/spiffe/spire/proto/spire-next/api/server/agent/v1"
+	"github.com/spiffe/spire/proto/spire-next/types"
+	"github.com/spiffe/spire/proto/spire/common"
+	"github.com/spiffe/spire/proto/spire/server/datastore"
 	"github.com/spiffe/spire/test/fakes/fakedatastore"
 	"github.com/spiffe/spire/test/spiretest"
+	"github.com/stretchr/testify/require"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
 )
 
+var agent1 = "spiffe://example.org/agent-1"
+var agent2 = "spiffe://example.org/agent-2"
+var agent3 = "spiffe://example.org/agent-3"
+
```

Comment:
nit: I don't feel too strongly about this but we typically group all the vars in one block:
```
var (
   var1 =
   var2 =
   var3 =

   ...
```